tests/subsys/pm/device_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(device_runtime_test)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
+
+target_include_directories(app PRIVATE
+	${ZEPHYR_BASE}/kernel/include
+	${ZEPHYR_BASE}/arch/${ARCH}/include)
```

Comment:
are we using any of these private includes here?

Reply:
this and the next comment is because I just copied the driver from another test.

Reply:
leftover from another test. Just removing it.